doc/02_Pattern sicurezza/05_accesso-del-fruitore/03_id_auth_rest_01.rst

```diff
@@ -111,7 +111,7 @@ Regole di processamento
 Note:
 
 -  Gli algoritmi da utilizzare in alg sono indicati al capitolo 7
```

Comment:
```suggestion
```

Reply:
@mfortini fai tu fix con: Gli algoritmi da utilizzare in alg sono indicati nelle Linee Guida sulla sicurezza, emanate dall'Agenzia per l'Italia Digitale ai sensi dell'articolo 71 del decreto legislativo 7 marzo 2005, n. 82 (Codice dell'Amministrazione Digitale).